telegram/ext/dispatcher.py

```diff
@@ -277,6 +303,17 @@ def process_update(self, update):
             try:
                 for handler in (x for x in self.handlers[group] if x.check_update(update)):
                     handler.handle_update(update, self)
+                    if self.persistence:
+                        if self.persistence.store_chat_data:
+                            try:
+                                self.persistence.update_chat_data(self.chat_data)
+                            except Exception:
```

Comment:
Is there a need to have separate exception handlers for chat/user data?

Reply:
I always learned to have as few statements in a `try` block as possible.